### DISCUSSION TITLE
mutate changes scroll position after mutating

### DISCUSSION THREAD

**Discussion by Mood-al** [vercel/swr#1525] [Q&A]
Hey guys ,
I'm facing an issue with mutate in SWR I'm using the mutate that exists inside useSWR hook and after updating the data as im showing in the picture the scroll position scrolls to down i tried many things to solve it but nothing worked!! 
 here when i replaced the res.data to first like so `  mutate([res.data, ...filteredReviews]);` it worked but if i tried to edit something in the middle same problem returns back

![image](https://user-images.githubusercontent.com/63562308/135670719-20a60130-9e04-496a-bd9a-1bb77e025678.png)

**Comment by shuding** [comment]
What is the "scroll position" you are referring to? Can you share the full code, or a minimal reproduction that we can take a look? The code from your screenshot looks good.

**Reply by Mood-al** [reply]
I solved it using map

![image](https://user-images.githubusercontent.com/63562308/136712186-893ced43-1c53-4c9a-9423-2a4364d211c9.png)
